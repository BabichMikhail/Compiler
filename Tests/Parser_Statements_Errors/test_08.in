var i, j : integer;
begin 
	try 
		for i := 0 to 100 do begin 
			for j := 0 to 100 do begin
			end
		end;;
	finally ;;
	end;
end;
end.