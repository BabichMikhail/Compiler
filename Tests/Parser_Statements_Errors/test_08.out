Fatal: Syntax error, "." expected but ";" found in Line 9 Column 4
